bot/exts/easter/save_the_planet.py

```diff
@@ -0,0 +1,28 @@
+import json
+import random
+from pathlib import Path
+
+from discord import Embed
+from discord.ext import commands
+
+
+class SaveThePlanet(commands.Cog):
+    """A cog that teaches users how they can help our planet."""
+
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+        self.embed_data = []
+
+        with open(Path("bot/resources/easter/save_the_planet.json"), 'r', encoding='utf8') as f:
+            self.embed_data = [embed for embed in json.load(f)]
```

Comment:
```suggestion
            self.embed_data = json.load(f)
```
I missed this last time, but since the loaded json is already a list, we can simply do this.